### ISSUE TITLE
readability of 7.8.1

### ISSUE THREAD

**Issue by fabianneuhaus** [tillmo/DOL#155]
use case 7.8.1 illustrates the problem very well. However, it is very hard to understand the solution. 
- it seems that there is a disconnect between the labels that are used in the diagram and the labels that are used in the formalisation. E.g., I wasn't able to figure out what "cd". The formalisation uses the following symbols: cd, atm, psm, csd, sqd. In the figure I find the labels sd, stm, cmp. 
- psm2cd, atm2cd  are undefined, abstract_to_concrete_atm will be defined only in the next use case 
- the intended semantics of the DOL statements (e.g. view cd2psm = cd to { psm hide along psm2cd} ) is not explained. 
In short, while the text sets up the problem very well, it does not explain the solution DOL offers enough. Because the DOL formulas are not self-explanatory, it would help if the DOL formulas would be explained.

**Comment by fabianneuhaus** [comment]
why is cd2int not included in the combine?

**Comment by fabianneuhaus** [comment]
I am a little confused about the "along atm2cd" notation. I always thought "along" is followed by a OMS language translation. I guess cd is class diagram (but I don't think most readers would guess that). But what the heck is atm?
